sycl/test-e2e/Graph/graph_record_threaded_record.cpp

```diff
@@ -0,0 +1,65 @@
+// REQUIRES: level_zero, gpu
+// RUN: %clangxx -pthread -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// Expected fail as thread safety not yet implemented
+// XFAIL: *
```

Comment:
This test can sometimes pass and sometimes fail, so I will investigate these threaded tests more, as in the emulation mode I also see fails in other threaded tests.